d2l-scroll-wrapper.html

```diff
@@ -296,36 +293,43 @@
 					'scroll': 'checkScrollThresholds'
 				},
 
-				handleTapBack: function() {
-					var scrollDistance = this.clientWidth * this.scrollAmount;
-					var isRTL = !!this.$$('[dir="rtl"] *');
-					if (isRTL) {
-						if (this.scrollRtlReverse) {
-							this.scroll(scrollDistance - this.scrollLeft, true);
-							return;
-						} else if (this.scrollRtlNegative) {
-							this.scroll(this.scrollLeft - scrollDistance, true);
-							return;
-						}
+				isRTL: function() {
+					if (this.$$('[dir="rtl"] *')) {
+						return true;
 					}
-					this.scroll(this.scrollLeft - scrollDistance, true);
+					try {
+						return !!this.$$(':host-context([dir="rtl"]) *');
+					} catch (e) {
+						return false;
+					}
+				},
+
+				/* Scrolls to the left. Right when dir=rtl */
+				handleTapLeft: function() {
+					var scrollDistance = this.clientWidth * this.scrollAmount;
+					this.scrollDistance(-scrollDistance, true);
 				},
 
-				handleTapForward: function() {
+				/* Scrolls to the right. Left when dir=rtl */
+				handleTapRight: function() {
 					var scrollDistance = this.clientWidth * this.scrollAmount;
-					var isRTL = !!this.$$('[dir="rtl"] *');
-					if (isRTL) {
+					this.scrollDistance(scrollDistance, true);
+				},
+
+				/* Scrolls the set distance. positive === right, negative === left. Reversed when dir=rtl */
+				scrollDistance: function(distance, smooth) {
+					if (this.isRTL()) {
```

Comment:
Probably not, but the WCT tests do. I can't think of any reason this would be done in a normal circumstance.

Thoughts @dlockhart ?